src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -26,6 +26,8 @@
 
 /** an object representing the Drivetrain of a swerve drive frc robot */
 public class Drivetrain extends SubsystemBase {
+  public static final Vector ORIGIN = new Vector();
```

Comment:
Should be private if it's going to live here.

Reply:
I think I'll go with emmy's suggestion to put it in the Vector class